content/WebAuthn/WebAuthn_Developer_Guide/Overview.adoc

```diff
@@ -30,7 +30,7 @@ FIDO2 credentials can provide MFA by employing user verification authorization g
 == Public key-based credentials
 In general, a credential is data that proves a person’s identity or qualification. This credential is used to authenticate to an online service, also known as a Relying Party (RP). FIDO2 uses public-key cryptography to authenticate users. An RP-specific credential key pair, i.e., a private key and a public key, is generated on the authenticator. The public key is sent to the RP at registration time. The private key never leaves the authenticator. When the user makes the request to login, the authenticator sends an assertion that proves the user possesses the private key. The RP uses the public key to validate the assertion before allowing the user to login.
 
-FIDO2 introduces the concept of a resident key credential, which is/are stored on the authenticator instead of encrypted and stored on the server. Resident keys also enable passwordless authentication scenarios, at the expense of consuming limited storage space on the authenticator.
+FIDO2 introduces the concept of a discoverable key credential, also know as a resident key, which is/are stored on the authenticator instead of encrypted and stored on the server. Discoverable keys also enable passwordless authentication scenarios, at the expense of consuming limited storage space on the authenticator.
```

Comment:
```suggestion
FIDO2 introduces the concept of a client-side discoverable credential, also know as a resident key, which is/are stored on the authenticator instead of encrypted and stored on the server. Discoverable keys also enable passwordless authentication scenarios, at the expense of consuming limited storage space on the authenticator.
```
The spec changed 'resident key' to 'client-side discoverable credential' and I suggest we do the same for consistency. The term 'discoverable key credential' may cause confusion.